doc/source/reference/arrays.dtypes.rst

```diff
@@ -179,19 +179,14 @@ Built-in Python types
     ===================      ===============
     :class:`int`             :class:`int\_`
     :class:`bool`            :class:`bool\_`
-    :class:`float`           :class:`float\_`
-    :class:`complex`         :class:`cfloat`
+    :class:`float`           :class:`double`
+    :class:`complex`         :class:`cdouble`
     :class:`bytes`           :class:`bytes\_`
     :class:`str`             :class:`str\_`
     :class:`memoryview`      :class:`void`
     (all others)             :class:`object_`
     ===================      ===============
 
-    Note that ``str_`` corresponds to UCS4 encoded unicode strings, while
-    ``string_`` is an alias to ``bytes_``. The name ``np.unicode_`` is also
-    available as an alias to ``np.str_``, see :ref:`Note on string
-    types<string-dtype-note>`.
```

Comment:
The note about `str_` being UCS4 encoded unicode strings should probably stay, a naive user might assume it's UTF8 encoded.

Reply:
Sure! I kept `str_` note.